core/module_specifier.rs

```diff
@@ -207,6 +208,18 @@ impl PartialEq<String> for ModuleSpecifier {
   }
 }
 
+impl Ord for ModuleSpecifier {
+  fn cmp(&self, other: &Self) -> Ordering {
+    self.0.cmp(&other.0)
+  }
+}
+
+impl PartialOrd for ModuleSpecifier {
+  fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+    Some(self.cmp(other))
+  }
+}
+
```

Comment:
Seems we could derive `PartialOrd` and `Ord` automatically instead of manually implementing them.

```rust
#[derive(Debug, Clone, Eq, Hash, PartialEq, serde::Serialize, PartialOrd, Ord)]
/// Resolved module specifier
pub struct ModuleSpecifier(Url);
```

I'd prefer this :)